lib/main/STM32_USB_OTG_Driver/inc/usb_defines.h

```diff
@@ -174,7 +174,9 @@
 #define HCCHAR_BULK                            2
 #define HCCHAR_INTR                            3
 
+#ifndef MIN
```

Comment:
Yeah, this one's good as well.

Reply:
Cool. In general I would prefer not to touch ST USB Stack as it's already very nasty and instead of fixing everything we might as well move to something more flexible like this: https://github.com/IntergatedCircuits/USBDevice

Reply:
Fair enough.